lib/content._coffee

```diff
@@ -276,12 +276,8 @@ module.exports = class Content
 
         else
             idPath = getFilePathForId id
-            urlPath = getFilePathForURL url
-            urlToIdPath =
-                Path.join DIR_PATH_FROM_URL_TO_ID, Path.basename idPath
-
             FS.writeFile idPath, content._stringifyData(), _
-            FS.symlink urlToIdPath, urlPath, _
+            # TODO: Update content-by-url in Cloud Files
```

Comment:
I figure we can just leave this unfixed at least until we start accepting new URLs (or maybe not ever if we go over to redis first)...at any rate this code isn't being run at this point.